src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/reduce.cpp

```diff
@@ -143,7 +144,8 @@ const auto params_OneAxis = testing::Combine(
             testing::Values(ElementType::undefined),
             testing::ValuesIn(inputShapes_dyn)),
         testing::Values(emptyCPUSpec),
-        testing::Values(emptyFusingSpec));
+        testing::Values(emptyFusingSpec),
+        testing::ValuesIn(additionalConfig()));
```

Comment:
Do we really need to check fp32/bf16/fp16 in each test instanse? I would prefer to avoid huge test cases growth. Can we analyse which scenarious are really worth (and enough) to cover and update only them?

Reply:
Good point! Removed the test cases that are not that necessarily needed. Thanks Dmitry!